api/endpoints.go

```diff
@@ -6,3 +6,6 @@ const GetAppsAPI = "/apps"
 // DeployByIDapi is the api string formatter for Sprintf(), first argument
 // app id, second argument deploy id
 const DeployByIDapi = "/app/%s/deploy/%s"
+
+// DeployAPI is the api string
+const DeployAPI = "/app/deploy"
```

Comment:
**suggestion** in order to prevent inconsistencies in naming (`DeployByIDapi`, `DeployAPI`), in a future MR WDYT about renaming these constants with the following format:

```
const APIDeployByID = ""
const APIDeploy = ""
```

Alternatively, we could also create a struct:

```
type APIEndpoints struct {
  Deploy: string,
  DeployByID: string
}

API := &APIEndpoints{
  Deploy: "/app/deploy",
  DeployByID: "/app/%s/deploy/%s"
}
```

and then use it across the application like:

```
stormkit.Post(API.Deploy);
```

But I'd suggest to do that in a separate MR in order to keep things separate 👍

Reply:
Done the second 9aaf786